cmd/translatesfx/translatesfx/discovery_rules_test.go

```diff
@@ -41,7 +42,11 @@ func TestDiscoveryRuleToRCRule_HasPort(t *testing.T) {
 	saRule := `target == "hostport" && has_port == true`
 	rcRule, err := discoveryRuleToRCRule(saRule, nil)
 	require.NoError(t, err)
-	program, err := expr.Compile(rcRule)
+	program, err := expr.Compile(rcRule,
```

Comment:
would it be better to make a helper function in some shared utils module for this? I see it ends being used many times

Reply:
I think that could work in some cases, but it's not always called with the same arguments (the usage in the [evaluator file](https://github.com/signalfx/splunk-otel-collector/pull/3559/files#diff-9c5b5ddac88f34b57f2f1604f128509bf4c51b183f0561a0b2c9dc4555044c6d) for example).

It's also just a single method call, so even though I agree with the thought of centralizing the logic, it seems like it might be over-complicating it.

Do you think it's still worthwhile?

Reply:
Looking at the usage, it seems like the extra param we may pass is env so we could just account for that until we find ourselves having to use more options. I don't feel too strongly about this and we could tackle it in a later PR.

Reply:
Sorry for my ignorance, but do you have a thought of where the best place would be? I was thinking `internal/common` made sense, but a few of the callers are in the signalfx-agent pkg, so I believe that means `internal` can't be the location.

Reply:
looks like we might need this change in translatesfx only